alonzo/impl/src/Cardano/Ledger/Alonzo/PParams.hs

```diff
@@ -321,12 +328,13 @@ encodePParamsUpdate ppup =
     !> omitStrictMaybe 12 (_d ppup) toCBOR
     !> omitStrictMaybe 13 (_extraEntropy ppup) toCBOR
     !> omitStrictMaybe 14 (_protocolVersion ppup) toCBOR
```

Comment:
was 15 the old parameter that we got rid of?

Reply:
yes